src/kitovu/sync/syncing.py

```diff
@@ -65,3 +73,19 @@ def start(connection_settings: ConnectionSettings) -> None:
             print(f'Local digest: {digest}')
 
     plugin.disconnect()
+
+
+def config_error(config_file: typing.Optional[pathlib.Path]) -> typing.Union[str, None]:
```

Comment:
Maybe rename this to `validate_config` or so?

Reply:
I've thought about that as well, but if I see a method `validate_config` I either don't expect a return value or a boolean.

Moving it to `settings.py` would lead to circular dependencies. Unless we would move `_find_plugin` to the `settings.py` as well, which is a bit unintuitive.